### DISCUSSION TITLE
Road surface not visible in "Bike mode"

### DISCUSSION THREAD

**Discussion by Nils-Berlin** [osmandapp/OsmAnd#11387] [Q&A]
Hey folks,
at first I want to say that I a BIG fan of OSMand and it's functionality, I use it a lot, mainly for Navigation while biking.
But there is one issue I can't solve:

If I use "Car mode" then I can nicely see the road surfaces on the map (grey dotted lines for asphalt, blue for cobblestone etc.). When I switch to "Bike Mode" then these markers disappear - even though Configure map -> Details -> Show road surface  is enabled. 

I really love this feature, especially for biking as there is a lot of nasty cobbelstone-roads in Berlin - what am I doing wrong?

Thanks for your help & best wishes

**Comment by tommycrock** [comment]
Interesting. I don't use it for the bike but do use the road surface
feature. I also can't get it to show in bike mode, but noticed the options
are different in bike mode than car/foot. I have fewer options but also the
surface one is worded differently "road surface integrity" instead of "show
road surface". I wonder if this might be referring to a smoothness tag or
tracktype perhaps?

**Reply by Nils-Berlin** [reply]
Hi Tommy,
Thanks for your efforts! That's strange, here the wording in Map rendering -> Details is always "Show road surface", no matter what mode (Bike/Car/Foot) is chosen. Also in all modes there is the option "Show road quality" right below that button.

![photo_2021-04-13_14-05-57](https://user-images.githubusercontent.com/82273666/114549635-890ca480-9c61-11eb-88d2-72503f65656f.jpg)

**Reply by intrigeri** [reply]
It does not depend on the mode (Bike/Car/Foot), but on the map style. The default ("OsmAnd") map style behaves as you're describing, while OP is likely using another map style, such as "Topo".
